tracer/src/Datadog.Trace/Iast/SensitiveData/EvidenceRedactor.cs

```diff
@@ -63,10 +66,10 @@ public bool IsKeySensitive(string? key)
         {
             return _keysRegex.IsMatch(key);
         }
-        catch (Exception err)
+        catch (RegexMatchTimeoutException err)
```

Comment:
This is the only exception type that could be thrown? If another was thrown it would be caught later and not leak into the client app?

Reply:
The rest of errors are catch in VulnerabilityBatch.cs and treated as regular errors, with Log.Error when they happen.